docs/serialization.md

```diff
@@ -53,7 +53,6 @@ A graph is serialized as an object containing an `attributes`, a `nodes` & an `e
   * <span class="code">object</span> `edges`: containing a list of serialized edges (can be omitted).
 
 ```js
-graph.addNodesFrom(['Thomas', 'Eric']);
```

Comment:
@nikelborm you need to replace this call with two `addNode` ones or use the `mergeEdgeWithKey` else the example is not correct because the nodes where not added.

Reply:
Repeated as you do it here: https://github.com/graphology/graphology/commit/9d08ec93c0492765990b84a012282e50485d4de4#diff-2b85528e08fa1511d8a1cb56775e93bd6ac90bd02d9c1583a6b6ceada57216dbR113